File: src/java/Boot/src/com/core/api/mcpe/level/LevelData.java
package com.core.api.mcpe.level;

import com.core.api.engine.PointerClass;

public class LevelData extends PointerClass {
    public LevelData(long ptr){
        super(ptr);
    }

    public void setSeed(int seed){
        setSeed(this.getPointer(), seed);
    }

    public int getSeed(){
        return getSeed(this.getPointer());
    }
    
    public static native void setSeed(long pointer,int seed);
    public static native int getSeed(long pointer);
}

File: src/native/main/core/mcpe/level/level_data.cpp
#include <level/LevelData.h>
#include <java.h>


export(void, mcpe_Level_LevelData_setSeed,jlong pointer,jint seed){
    return ((LevelData*)pointer)->setSeed((unsigned int) seed);
}

export(jint,mcpe_Level_LevelData_getSeed,jlong pointer){
    return (jint) ((LevelData*) pointer)->getSeed();
}